examples/fabric-sync/shell/ShellCommands.cpp

```diff
@@ -45,6 +46,7 @@ static CHIP_ERROR PrintAllCommands()
     streamer_printf(sout,
                     "  add-device       Pair a device to local fabric. Usage: app add-device node-id setup-pin-code "
                     "device-remote-ip device-remote-port\r\n");
+    streamer_printf(sout, "  pair-device       Pair a device to local fabric. Usage: app pair-device node-id code\r\n");
```

Comment:
super nit. Also do we want to mention that is is a device that is already onnetwork?
```suggestion
    streamer_printf(sout, "  pair-device      Pair a device to local fabric. Usage: app pair-device node-id code\r\n");
```